apps/functions/applications-migration/common/migrators/s51-advice-migration.js

```diff
@@ -81,9 +86,11 @@ export async function migrateS51AdviceForCase(log, caseReference) {
 const mapStatus = (status) =>
 	({
 		checked: 'checked',
```

Comment:
are these the values to be stored in our DB?  These look more like the schema values.  Our DB values are listed in the mapS51AdviceStatusToSchemaStatus fn where they are mapped to the schema values for schema payload.  But in this case should they not be: checked, not_checked, ready_to_publish, published, or donotpublish ?

Reply:
If I remember correctly, this is to correct some values coming from ODW that do not conform to the schema. This mapping should be done in ODW's curated layer, which is covered in a ticket on their board: https://pins-ds.atlassian.net/browse/ODW-1122.